packages/keystone/bin/utils.js

```diff
@@ -7,6 +7,48 @@ const path = require('path');
 
 const { DEFAULT_ENTRY, DEFAULT_PORT } = require('../constants');
 
+const verifyTableMessages = verifyTableResults => {
+  const verifyTableErrors = verifyTableResults.filter(({ isRejected }) => isRejected);
+
+  if (Array.isArray(verifyTableErrors)) {
```

Comment:
This `if` statement is redundant. `verifyTableErrors` is the result of `.filter(), which will always return an array. Did you want instead to check `verifyTableErrors.length`?

There's probably no need for that even, you can just do the `verifyTableErrors.forEach()` and if there are no errors then you'll have a no-op.